app/components/ui/tale-tab-metadata/template.hbs

```diff
@@ -0,0 +1,91 @@
+<div class="ui form">
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Title</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.title placeholder="Enter a title..." readonly=cannotEditTale}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Authors</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.authors placeholder="Enter the author's name..." readonly="true"}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Category</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.category placeholder="Choose a category..."  readonly=cannotEditTale}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Environment</label>
+    <div class="field thirteen wide column">
+      <div class='ui icon selection dropdown {{if (eq environments.length 0) "loading"}}' style='width:100%'>
+        <input name='TaleImageId' type='hidden' onchange={{action "setTaleEnvironment" value="target.value"}}>
+        <i class='dropdown icon'></i>
+        <div class='default text'> Choose an environment...</div>
+        <div class='menu'>
+          {{#each environments as |env|}}
+            <div class='item' data-value={{env._id}}>
+              <img class='ui image' src={{env.icon}} style='height:28px;width:28px;'> {{env.name}}
```

Comment:
Favor classes usage over inline style

Reply:
Abstracted all inline styles to `app/styles/tale-tabs.scss` until I know of a more acceptable place to put them